composer.json

```diff
@@ -22,6 +22,6 @@
 	},
 	"require": {
 		"composer-plugin-api": "^1.0",
-		"composer/installers": "~1.0.22"
+		"composer/installers": "^1.1"
```

Comment:
I think I'd prefer `^1.0` in case there are any other dependencies that require a more specific version of `composer/installers` (whether warranted or not) because that would prevent the solver from finding a set of compatible packages